tools/UpdatePaths/76732_multi-tile_airlocks.txt

```diff
@@ -0,0 +1,4 @@
+#comment Repathing for new multi-tile airlocks.
+
+/obj/machinery/door/airlock/multi_tile/glass : /obj/machinery/door/airlock/multi_tile/public/glass{@OLD}
+/obj/machinery/door/airlock/glass_large : /obj/machinery/door/airlock/multi_tile/public/glass{@OLD}
```

Comment:
This file seems to be a duplicate. It shouldn't be in this folder, and there's a near identical copy in the Scripts folder

Reply:
Oop yeah, I forgot to delete it after running updatepaths.